CHANGELOG.md

```diff
@@ -8,9 +8,16 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.0.0-beta.7] - 2019-04-17
+
+### Changed
+
+- Update French and Canadian French `.po` files with the latest translations.
+
 ### Fixed
 
-- Fix a display glitch with parent filters when active
+- Fix broken `hub` job in CI publishing the Docker image,
```

Comment:
Not sure CI related changes should be logged here. It's not about the app but its building environment.